### DISCUSSION TITLE
Problem in implementing Versioning if tags are included

### DISCUSSION THREAD

**Discussion by KarthikModelon** [squidfunk/mkdocs-material#4239] [Q&A]
I am facing a issue when implementing versioning. While serving mike it shows an error "No module named 'tags'
" .If I remove the tags from yml file mike is serving properly. Is it not possible to implement both tags and version? If possible, please help me out how to do that.
![image](https://user-images.githubusercontent.com/109127978/184135752-6e66d498-7b92-4929-819a-261f60bb6add.png)

**Comment by squidfunk** [comment]
Please provide a minimal reproducible example and attach it here as a zip with steps to reproduce what you're seeing.
